sklearn/metrics/tests/test_score_objects.py

```diff
@@ -121,6 +127,24 @@ def predict(self, X):
         return self.y
 
 
+class ClassifierWithFitAndPredict(BaseEstimator, ClassifierMixin):
```

Comment:
I think you could replace this with a [DummyClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html) with `constant=1.0`.